### PR TITLE
Ballio improvements

### PR DESCRIPTION
**Pull request by c0d3rman**
Centralized Ballio constants and refactored some stuff.

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/commands/BallioCycle.java [hunk 1]
```diff
@@ -10,8 +11,9 @@ public BallioCycle() {
 		super("BallioCycle");
 		requires(RobotMap.Component.ballIO);
 		addSequential(new BallioDoorSetOuttake());
-		addParallel(new MotorConstant(RobotMap.Component.ballIO.directionalRoller, 1.0));
-		addParallel(new MotorConstant(RobotMap.Component.ballIO.elevatorAndIntakeRoller, 1.0));
-		addSequential(new MotorConstant(RobotMap.Component.ballIO.hopperRollers, 1.0));
+		addParallel(new MotorConstant(RobotMap.Component.ballIO.directionalRoller, BallIO.DIRECTIONAL_ROLLER_INTAKE_SPEED));
```
Comment (oldgalileo) [inline src/org/usfirst/frc4904/robot/commands/BallioCycle.java:14]: Good use of constants.

File: src/org/usfirst/frc4904/robot/commands/BallioDoorSet.java [hunk 2]
```diff
@@ -15,7 +15,7 @@ public BallioDoorSet(BallIO.DoorState targetState) {
 
 	@Override
 	protected void initialize() {
-		RobotMap.Component.ballIO.doorSet(targetState);
+		RobotMap.Component.ballIO.setDoorState(targetState);
```
Comment (oldgalileo) [inline src/org/usfirst/frc4904/robot/commands/BallioDoorSet.java:18]: Good rename.

File: src/org/usfirst/frc4904/robot/commands/BallioIntake.java [hunk 3]
```diff
@@ -10,8 +11,9 @@ public BallioIntake() {
 		super("BallioIntake");
```
Comment (oldgalileo) [inline src/org/usfirst/frc4904/robot/commands/BallioIntake.java:11]: These could probably be combined since the only thing that changes is the values.

**Review by oldgalileo — APPROVED** [review]
Looks good. Little things could probably be made better but I'm not sure how.

**Review by ajnadel — APPROVED** [review]
DO IT

**Review by ajnadel — APPROVED** [review]

**Pull request merged**